entries/r/rippling.com.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Rippling": {
+    "domain": "rippling.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://help.rippling.com/s/article/1500000619442",
```

Comment:
This page is only accessible with a login. Could you provide some other documentation or screenshots of the set-up process? (You should redact any personal information there.)

Reply:
<img width="858" alt="image" src="https://user-images.githubusercontent.com/279376/157581127-4881623f-ce60-426b-af89-9df6d584215d.png">